### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by PabRod**
In the process of making our package CRAN-worthy, I made some changes worth merging to master. The most relevant of them is the addition of the `ifadv` dataset as a suggested dependency.

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ sonar.project.properties
 ^working_example.R
 
 notepad.R
```
Comment (bvreede) [inline .Rbuildignore:8]: Do these need to be there? (I know this is not part of this PR, but sonarcloud is not used anymore, and those two .R files should also not be part of the package, so can we remove these too?)

File: .github/workflows/R-CMD-check.yaml [hunk 2]
```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - 61-cran
```
Comment (bvreede) [inline .github/workflows/R-CMD-check.yaml:7]: This should be removed still; we don't want to clutter up the actions file.

File: DESCRIPTION [hunk 3]
```diff
@@ -35,4 +32,10 @@ Imports:
     tidyselect,
     tibble,
     viridis
+Suggests: 
+    devtools,
```
Comment (bvreede) [inline DESCRIPTION:36]: This one is only recommended if someone would install ifadv via devtools, right? But then, if we put ifadv in Suggests as well, we no longer need devtools...

**Review by bvreede — APPROVED** [review]

File: .github/workflows/R-CMD-check.yaml (hunk 2 above)
Comment (PabRod) [reply .github/workflows/R-CMD-check.yaml:7]: I'll keep using the branch at least until submission. I'll remove it afterwards.

File: DESCRIPTION (hunk 3 above)
Comment (PabRod) [reply DESCRIPTION:36]: Sounds good!
```suggestion
```

File: .Rbuildignore (hunk 1 above)
Comment (PabRod) [reply .Rbuildignore:8]: Let's remove them.

```suggestion
```

File: .github/workflows/R-CMD-check.yaml (hunk 2 above)
Comment (PabRod) [reply .github/workflows/R-CMD-check.yaml:7]: Second thoughts, let's use a draft pull request to trigger this more cleanly.

```suggestion
```

**Pull request merged**